src/client/components/Home/Home.css

```diff
@@ -75,19 +75,23 @@
 }
 .sponsors {
 	padding-top:50px;
-	padding-bottom:50px;
+	padding-bottom:60px;
 	margin:0px;
 	margin-left:auto;
 	margin-right:auto;
 	width:50%;
 	justify-content: center;
+	text-align:center;
 }
 .sponsor {
 	padding:0px;
-	padding-top:20px;
+	padding-top:30px;
```

Comment:
space after rule key, after padding-top: 
This comment refers to all css rules without space after key